### DISCUSSION TITLE
how to use transform function to insert username and password

### DISCUSSION THREAD

**Discussion by eudjinnl** [nornir-automation/nornir#800] [Q&A] [answered]
Can't understand how to use transform function to insert username and password for hosts.
I use the last version of nornir. SimpleInventory. I use config file to initialize nornir.

**Comment by ubaumann** [comment, marked as answer]
Nornir-utils has an example implementation and some documentation

https://github.com/nornir-automation/nornir_utils/blob/master/docs/source/tutorials/load_credentials.ipynb

**Reply by eudjinnl** [reply]
Thank you. It helped a lot!
